site/content/internal/sustained-engineering.md

```diff
@@ -81,9 +81,19 @@ SET attends all triage meetings. During triage, SET should:
 
 During non-release weeks, triage is held twice a week. Near the end of the release cycle, triage is held daily.
 
-## Support Responses
+## Customer and Pre-Sales Support Escalation Process
 
-Part of SET's responsibility is to interface with the customer support team. SET's primary goal in respect to support is to triage escalated support issues, work on any high priority customer bugs requiring code change, as well as provide training and knowledge share with the customer support team. To help us do this better, SET members should use the following responses when receiving issue escalations from the customer support team:
+Part of SET's responsibility is to interface with the customer support team. SET's primary goal in respect to support is to triage escalated support issues, work on any high priority customer bugs requiring code change, as well as provide training and knowledge share with the customer support team. When the pre-sales and customer support teams require engineering support, they should:
+
+1. Post the issue in [Sustained Engineering](https://community.mattermost.com/core/channels/sustained-engineering) channel. If the message contains confidential information about a customer, then post in [Customer Support](https://community.mattermost.com/private-core/channels/community) channel in private Staff team.
+2. At-mention SET team lead, identified as the first person in the [Sustained Engineering](https://community.mattermost.com/core/channels/sustained-engineering) channel header.
+3. If no prompt solution is presented, create a ticket in [Mattermost Jira project](https://mattermost.atlassian.net/) and assign "Sustained Engineering" as the **Mattermost Team**. The ticket will be reviewed by a SET team during Tuesday or Thursday triage.
```

Comment:
```suggestion
3. If no prompt solution is presented, create a ticket in [Mattermost Jira project](https://mattermost.atlassian.net/) and assign "Sustained Engineering" as the **Mattermost Team**. The ticket will be reviewed by SET team during Tuesday or Thursday triage.
```

Reply:
Still think that they should be present in the meeting when the ticket is going to get triaged

Reply:
FYI that an SA team member tried to join triage meetings this week, but on Tuesday the call was occupied by release discussions (and they had to hop out after 30 mins), on Thursday the Zoom meeting they joined was empty / no one was there.

Not saying we shouldn't have the members join the call, but propose we make it clear how support/pre-sales team participates, and what the fallback is if they can't join (SAs are often travelling to customer sites)

Reply:
@enahum 

I'm sure if support wants urgent attention they themselves will attend triage and provide feedback. As @jasonblais points out, it might become difficult for them to attend sometimes, and no point in adding something to the process if it's not given any weight.

Lets proceed as is and see how things work out. We can always refine this process further by encouraging them to attend at any time if we deem it necessary :)

Reply:
```suggestion
If no solution or next steps (such as a Jira ticket) are presented within 24 hours, create a ticket in [Mattermost Jira project](https://mattermost.atlassian.net/) and assign "Sustained Engineering" as the **Mattermost Team**. The ticket will be reviewed by a SET team during Tuesday or Thursday triage. If you need a resolution sooner than triage time, escalate to @chris.overton on https://community.mattermost.com.
```